react-components/src/context/UserContext.js

```diff
@@ -114,11 +128,14 @@ const UserContextProvider = props => {
         },
         token: userCookie,
         isSignedIn: isSignedIn(),
+        isAccountDropdownOpen: false,
         signInError: null,
         inProgress: false,
         createAccountError: null,
         createAccountEmail: null,
-        cartId: null
+        cartId: null,
+        accountDropdownView: 'SIGN_IN',
+        accountContainerQuerySelector
```

Comment:
@nettree Somehow I missed this, sorry for the late feedback: I don't think the `accountContainerQuerySelector` belongs in the user state. This is more like an "application state" element.

We should create an `ApplicationContext` that wraps all the app and makes this configuration available. I'll create a github issue.

Reply:
Yea, I agree, I was struggling a bit with this as well but can't find a more suitable place at that time, `ApplicationContext` sounds good, do you want me to add it? Or we want to deal with it later?

Reply:
We'll dewl with that in a separate issue